outputs.tf

```diff
@@ -0,0 +1,7 @@
+output "webapp_url" {
+    value = "https://${azurerm_app_service.main.default_site_hostname}"
+}
+
+output "cd_webhook" {
+    value = "https://${azurerm_app_service.main.site_credential[0].username}:${azurerm_app_service.main.site_credential[0].password}@${azurerm_app_service.main.name}.scm.azurewebsites.net/docker/hook"
+    }
```

Comment:
This contains a password. So you should set sensitive = true to avoid it being printed out in the CD pipeline's logs